app/controllers/reactions_controller.rb

```diff
@@ -95,9 +95,9 @@ def create
     render json: { result: result, category: category }
   end
 
-  def cached_user_positive_reactions(user)
-    Rails.cache.fetch("cached_user_reactions-#{user.id}-#{user.positive_reactions_count}", expires_in: 24.hours) do
-      user.reactions.positive
+  def cached_user_public_reactions(user)
+    Rails.cache.fetch("cached_user_reactions-#{user.id}-#{user.public_reactions_count}", expires_in: 24.hours) do
+      user.reactions.public_category
```

Comment:
Pointing this section out, since this cache was previously using on `.positive` and therefore the reason I'm keeping the `.positive` scope.